open-company/open-company-web
# Comment 😀 reactions

Card: https://trello.com/c/6FWiIfWK

### Review with https://github.com/open-company/open-company-interaction/pull/33

To test:
- with user A add a comment and post it with the Send button
- [ ] with user A add another comment, use Enter while typing to add a new line
- [ ] now post it with CMD+Enter
- now add some more mixed comments from user A and B
- [ ] user A can edit/delete only his own comments?
- [ ] user B can edit/delete only his own comments?
- [ ] delete a comment work?
- edit comment is not working right now: medium-editor is instantiated but destroyed right after (save doesn't work because of it)
- share a comment button is in the code but commented out until we have comments addressability
- [ ] user A can react to one of his own comments from the picker?
- [ ] user B can react to one of his own comments from the picker?
- [ ] user A can react to one of his own comments using a reaction left from B
- [ ] user B can react to one of his own comments using a reaction left from A
- [ ] user A can remove a reaction from a comment?
- [ ] user A can react to the post with the picker?
- [ ] user B can react to the post with the same reaction left by B?
- check the feedback items left in the Trello card to see if they are all fixed
- test comments any other way you can think of
- test reactions any other way you can think of
- test comment reactions any other way you can think of
